#177: Add test for duplicate instance name (#178)
exasol/extension-manager: doc/changes/changes_0.5.11.md

# Extension Manager 0.5.11, released 2024-05-07

Code name: Improve error message for creating duplicate instances

## Summary

> This release adds a new test to the shared integration test class `AbstractVirtualSchemaExtensionIT` that verifies the error message in case the user tries to create a virtual schema that already exists. It also allows overriding the parameter name of the virtual schema. This is required in case an extensions is not based on the base virtual schema extension.

Reviewer: ```suggestion
This release adds a new test to the shared integration test class `AbstractVirtualSchemaExtensionIT` verifying the error message in case the user tries to create a virtual schema that already exists. It also allows overriding the parameter name of the virtual schema. This is required in case an extension supports this parameter while not inheriting from the base virtual schema extension.
```
